tests/QE/test_xattrs.py

```diff
@@ -372,3 +374,434 @@ async def test_purge(self, cblpytest: CBLPyTest, dataset_path: Path) -> None:
 
         await sg.delete_database(sg_db)
         cbs.drop_bucket(bucket_name)
+
+    @pytest.mark.asyncio(loop_scope="session")
+    async def test_sg_sdk_interop_unique_docs(
+        self, cblpytest: CBLPyTest, dataset_path: Path
+    ) -> None:
+        sg = cblpytest.sync_gateways[0]
+        cbs = cblpytest.couchbase_servers[0]
+        num_docs = 10
+        num_updates = 10
+        sg_db = "db"
+        bucket_name = "data-bucket"
+
+        self.mark_test_step("Create bucket and default collection")
+        cbs.drop_bucket(bucket_name)
+        cbs.create_bucket(bucket_name)
+
+        self.mark_test_step("Configure Sync Gateway database endpoint")
+        db_config = {
```

Comment:
In the test I see the document content has the field `sdk` and `sg`, however I do not see any sync function that would be assigning these documents to the appropriate channels. Therefore all the documents are present in the all docs chanenl and the documents are then fetched via the Admin user that has access to the all docs chanenl. Another thing I noticed is that the user `vipul` is not being used to fetch the documents.